analyzers/src/ITs.JsonParser/Json/RuleIssues.cs

```diff
@@ -30,16 +30,19 @@ public class RuleIssue
 {
     public string Id { get; set; }
     public string Message { get; set; }
-    public SarifLocationFile Location { get; set; } // TODO: This should not re-use SarifLocationFile type.
+    public string Uri { get; set; }
+    public string Location { get; set; }
 
     public RuleIssue(SarifIssue issue)
     {
         Id = issue.RuleId;
         Message = issue.Message;
-        Location = new()
+        Uri = issue.NormalizedUri();
```

Comment:
Not related to this line: We need to make the output json in `actual` sorted by Uri and then by Location, then by message to produce deterministic order.

Otherwise local runs could produce differences, making a lot of noise in GIT that would be hard to work with